sdk/servicebus/service-bus/src/util/atomXmlHelper.ts

```diff
@@ -343,18 +343,17 @@ function setName(entry: any, nameProperties: any): any {
  * @param response
  */
 export function buildError(errorBody: any, response: HttpOperationResponse): RestError {
-  const errorProperties = errorBody.Error || errorBody.error || errorBody;
   let errorMessage;
 
   if (typeof errorBody === "string") {
     errorMessage = errorBody;
   } else {
-    errorMessage = errorProperties.Detail || errorProperties.detail;
+    errorMessage = errorBody.Error.Detail;
```

Comment:
Based on https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/ManagementClient.cs#L969, shouldn't this be `errorBody.Detail`?

Reply:
Based on test runs, looks like the Detail is present on here

Reply:
It could be possible that `errorBody.Error` does not exist. Then you would get a run time error here.
It could be possible that `errorBody.Error.Detail` does not exists. Then you would get an error with no message.